std/meta.d

```diff
@@ -955,15 +955,18 @@ unittest
 }
 
 
-// : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : //
-package:
-
-/*
- * With the builtin alias declaration, you cannot declare
- * aliases of, for example, literal values. You can alias anything
- * including literal values via this template.
+/**
+ * With the builtin alias declaration, you cannot declare aliases of,
```

Comment:
Is this a phobos thing or a personal preference?

I can't wait for the day when I can write "you" in an academic paper and people won't say "that doesn't make you sound self-important enough, you're making the rest of us look bad" :)